client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -52,8 +52,8 @@
             });
     }
 
-    CardsService.$inject = ['api', 'search', 'session'];
-    function CardsService(api, search, session) {
+    CardsService.$inject = ['api', 'search', 'session', 'desks'];
```

Comment:
Love to do that. But, I have no idea what exactly it does. We should ask someone who created or did major contribution to monitoring.